### PR TITLE
[GH Actions] Added linux build with clang 12

### PR DESCRIPTION
**Pull request by eponomar**

### CODE REVIEW

File: .github/workflows/ubuntu.yml [hunk 1]
```diff
@@ -6,6 +6,70 @@ env:
   CFLAGS: -O2 -Wformat -Wformat-security -Wall -Werror -D_FORTIFY_SOURCE=2 -fstack-protector-strong
 
 jobs:
+  clang12-api-latest:
```
Comment (dvrogozh) [inline .github/workflows/ubuntu.yml:9]: @onabiull : do we need to be over paranoid and have both clang-11 and clang-12 at the same time? I initially thought to just switch clang-11 configuration to clang-12...

**Review by dvrogozh — CHANGES_REQUESTED** [review]

File: .github/workflows/ubuntu.yml (hunk 1 above)
Comment (eponomar) [reply .github/workflows/ubuntu.yml:9]: In this case we need remove clang 10 as well...
I thought that we only ADD builds when found it.

Comment (dvrogozh) [reply .github/workflows/ubuntu.yml:9]: After offline talk: we'll go with modifying clang-11 configuration

Comment (eponomar) [reply .github/workflows/ubuntu.yml:9]: DONE

**Review by dvrogozh — APPROVED** [review]

**Review by dmitryermilov — APPROVED** [review]

**Pull request merged**